### PR TITLE
Add back AddRemoveRedirects script with commander

### PR DESCRIPTION
**Pull request by LizBaker**
The script can now be run like so:

`yarn addRedirects src/content/docs/accounts/accounts-billing`
or
`yarn removeRedirects src/content/docs/accounts/accounts-billing`

this uses the relative path of a given docs content folder

### CODE REVIEW

File: scripts/handlers.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+/* eslint-disable no-console */
+const { globSync } = require('glob10');
+const frontmatterGH = require('@github-docs/frontmatter');
```
Comment (sunnyzanchi) [inline scripts/handlers.js]: we have a `scripts/utils/frontmatter.js` module that can handle parsing frontmatter from a file. it doesn't have a stringify currently, but i'd rather we add it there than pull in another package

**Review by sunnyzanchi — COMMENTED** [review]
right now the interface is `yarn addRedirects` or `yarn removeRedirects`. i think it might be a little nicer if it was `yarn redirects add` or `yarn redirects remove`. we'd leave it open for future operations, and `yarn redirects --help` would show the help for both. @LizBaker what do you think?

**Review by clarkmcadoo — DISMISSED** [review]

File: scripts/handlers.js (hunk 1 above)
Comment (LizBaker) [reply scripts/handlers.js]: ooh gotcha, this is just from the original script

File: scripts/README.md [hunk 2]
```diff
@@ -66,3 +66,33 @@ It takes no arguments and will complete in about ~20min. It was created to be ru
...
+
+#### Example:
+
+`yarn redirects add src/contents/docs/taco`
+
+the file at path src/contents/docs/taco/cheese.mdx will now have a new redirect
+
+```
+redirects:
+- src/contents/docs/taco/cheese.mdx
```
Comment (sunnyzanchi) [inline scripts/README.md:88]: small note on these two redirects, redirects have slightly different paths
```suggestion
- /docs/taco/cheese
```

File: package.json [hunk 3]
```diff
@@ -175,6 +175,7 @@
   "license": "Apache-2",
   "scripts": {
     "add-files-to-translate": "node scripts/actions/add-files-to-translation-queue.js",
+    "addRedirects": "node scripts/addRemoveRedirects addRedirects",
```
Comment (sunnyzanchi) [inline package.json:178]: did we still want to keep this?

**Review by sunnyzanchi — APPROVED** [review]

File: scripts/README.md (hunk 2 above)
Comment (LizBaker) [reply scripts/README.md]: whoops!

File: package.json (hunk 3 above)
Comment (LizBaker) [reply package.json]: def not

**Review by sunnyzanchi — APPROVED** [review]

**Pull request merged**